hack/openshift/fetch-tektoncd-catalog-tasks.sh

```diff
@@ -53,6 +53,28 @@ declare -A TEKTON_CATALOG_TASKS=(
   # ["buildah"]="0.1"
   # ["openshift-client"]="0.2"
 )
+declare -r TEKTON_ECOSYSTEM="https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog"
+declare -A TEKTON_ECOSYSTEM_TASKS=(
+  ['task-buildah']="0.3.1"
+  ['task-git-cli']="0.3.0"
+  ['task-git-clone']='0.3.0'
+  ['task-kn-apply']='0.1.0'
+  ['task-kn']='0.1.0'
+  ['task-maven']="0.2.1"
+  ['task-opc']="0.1.0"
+  ['task-openshift-client']="0.1.0"
+  ['task-s2i-dotnet']='0.3.1'
+  ['task-s2i-go']='0.3.1'
+  ['task-s2i-java']='0.3.1'
+  ['task-s2i-nodejs']='0.3.1'
+  ['task-s2i-perl']='0.3.1'
+  ['task-s2i-php']='0.3.1'
+  ['task-s2i-python']='0.3.1'
+  ['task-s2i-ruby']='0.3.1'
+  ['task-skopeo-copy']='0.3.1'
+  ['task-tkn']='0.1.0'
+  ['task-buildpacks']='0.1.0'
```

Comment:
I don't think we should ship task-buildpacks because today we don't have such tasks in ClusteTasks cc @vdemeester